package-lock.json

```diff
@@ -627,7 +627,7 @@
             "dependencies": {
                 "buffer": {
                     "version": "4.9.1",
-                    "resolved": "https://registry.npmjs.org/buffer/-/buffer-4.9.1.tgz",
+                    "resolved": "http://registry.npmjs.org/buffer/-/buffer-4.9.1.tgz",
```

Comment:
I’m not sure why there are changes to `package-lock.json`? Can they be reverted? Maybe revert them and then run `npm ci` to make sure you are back to the default package install?